src/com/gfredericks/test/chuck/clojure_test/impl.cljc

```diff
@@ -0,0 +1,29 @@
+(ns com.gfredericks.test.chuck.clojure-test.impl
+  (:require #?(:clj  [clojure.test :refer [is]]
+               :cljs [cljs.test :refer-macros [is]])))
+
+;; copied from clojure.test.check, which privatized the function in
+;; recent versions.
+;;
+;; I think there might be plans for test.check to abstract this logic
+;; into a protocol or something, so I'm not too bothered by the
+;; copypasta for now.
+(defn ^:private not-falsey-or-exception?
+  [value]
+  (and value (not (instance? #?(:clj  Throwable
+                                :cljs js/Error)
```

Comment:
Thanks for pointing this out. I think you are correct. Related to that, we are not being able to report a `:fail`ed test due to an exception in an assertion as a distinct result from an `:error`ed test due to an exception not in an assertion. That can be seen in [our test](https://github.com/gfredericks/test.chuck/pull/15/files#diff-2eaa93a3b0a438ecb9060dfbfb3a393cR33). It seems to have its roots in [test.check not distinguishing between failure and error](https://github.com/clojure/test.check/blob/master/src/main/clojure/clojure/test/check/clojure_test.clj#L147-L153), which is not a trivial topic given shrinking is involved.